jsonschema-hyperschema.xml

```diff
@@ -1188,7 +1189,17 @@ GET /foo/
                     </author>
                     <date year="2016" month="October"/>
                 </front>
-                <seriesInfo name="Internet-Draft" value="draft-wright-json-schema-00" />
+                <seriesInfo name="Internet-Draft" value="draft-wright-json-schema-validation-01" />
```

Comment:
This change looks wrong. The "json-schema" reference points to the "core" document not the "validation" one. Copy/paste error?